extensions/guacamole-auth-openid/src/main/java/org/apache/guacamole/auth/openid/token/TokenValidationService.java

```diff
@@ -55,24 +59,24 @@
     @Inject
     private NonceService nonceService;
 
+    private static final String MSG_REJECTED_TOKEN = "Rejected OpenID token with malformed claim: {}";
+    private static final String MSG_MALFORMED_CLAIM = "Malformed claim within received JWT.";
```

Comment:
What's the reasoning behind moving these log messages into constants?

I'm wary of this because:

* The usage of `MSG_REJECTED_TOKEN` must be aware of the fact that it uses parameter substitution with `{}` (as well as the context of that `{}`). Likewise, the usage of `MSG_MALFORMED_CLAIM` must be aware that parameter substitution is _not_ used.
* The constants seem unlikely to be reused.
* The human-readable message can sometimes be useful when reading the source, as they provide context, but they are no longer readily readable in the vicinity of the relevant code.

Reply:
The rational is that multiple occurences of the same String should be "factorized" so it is all consistent throughout its use.
I can revert the previous state if you prefer.